core/com/lowagie/text/xml/XmlParser.java

```diff
@@ -63,23 +63,26 @@
 
 import com.lowagie.text.DocListener;
 import com.lowagie.text.ExceptionConverter;
+import com.lowagie.text.XMLUtil;
 
 /**
  * This class can be used to parse an XML file.
  */
 
 public class XmlParser {
-    
+
 /** This is the instance of the parser. */
     protected SAXParser parser;
-    
+
 /**
  * Constructs an XmlParser.
  */
-    
+
     public XmlParser() {
         try {
-            parser = SAXParserFactory.newInstance().newSAXParser();
+        	SAXParserFactory spf = SAXParserFactory.newInstance();
```

Comment:
if that would be a forked codebase I would probably `parser = prefer XMLUtil.newSAXParser()` but I guess this version is fine, in this context.

Reply:
Indeed a better design pattern is possible, but in the circumstances I wanted to keep the changes to the original code as small as they can be.